cli/installations.go

```diff
@@ -138,6 +138,25 @@ func (i *Installations) AddInstallation(ctx *GlobalContext, installPath string,
 	return installation, nil
 }
 
+func (i *Installations) DeleteInstallation(path string) error {
+	var idxToDelete = -1
+	for i, install := range i.Installations {
+		if install.Path == path {
+			idxToDelete = i
+			break
+		}
+	}
+
+	if idxToDelete < 0 {
+		return fmt.Errorf("installation with path %s does not exist", path)
+	} else {
+		copy(i.Installations[idxToDelete:], i.Installations[idxToDelete+1:])
+		i.Installations = i.Installations[:len(i.Installations)-1]
```

Comment:
Please use slice appending instead of copying.

i.e.

```go
i.Installations = append(i.Installations[:idxToDelete], i.Installations[idxToDelete+1:]...)
```